aerleon/lib/naming.py

```diff
@@ -840,10 +867,12 @@ def _ParseYamlNetworks(self, file_data: Dict[str, Any], file_name: str) -> None:
                     f'\nMultiple definitions found for network: {symbol}'
                 )
 
-            unit = _ItemUnit(symbol)
+            addr_unit = _ItemUnit(symbol)
+            fqdn_unit = _ItemUnit(symbol)
 
             # TODO(jb) This operation should be performed on the IR so we can hoist it from _ParseLine
```

Comment:
Might be best to delete this comment since the addition of fqdn means that _ParseYamlNetworks no longer mirrors _ParseLine.

Reply:
Done